staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -1331,3 +1348,13 @@ func isClientTimeout(err error) bool {
 	}
 	return false
 }
+
+func isStreamReset(err error) bool {
+	if err == nil {
+		return false
+	}
+	if urlErr, ok := err.(*url.Error); ok {
+		return strings.Contains(urlErr.Err.Error(), "INTERNAL_ERROR")
```

Comment:
hmm - INTERNAL_ERROR doesn't have to imply stream reset, right?

Reply:
Yeah, I am really unhappy about this, but I think there is not much better I can do in this repo now because the error originates elsewhere and what the client gets is not very distinctive.  Perhaps we should raise an issue on the http2 code?  Where exactly is that nowadays?

Reply:
That's unfortunate.
I would like the test to be fixed, so don't want to block the PR on improving it, but maybe a TODO here to improve that would help a future reader.

Reply:
In the force-push to 1f450695ffd I added a comment here.